Shadan/style.css

```diff
@@ -0,0 +1,159 @@
+*
+{
+    margin:0;
+    padding:0;
+}
+html{
+    background-color: #f7f7f7;
+}
+header{
+    background-color: #e6e6e6;
+
+}
+
+header .topcenter{
+    position:relative;
```

Comment:
I think that this is not useful here even if you remove it nothing will change 
It makes different only of the direct child of it have position absolute